Content.Server/Cargo/Systems/CargoSystem.Orders.cs

```diff
@@ -130,6 +131,20 @@ private void OnApproveOrderMessage(EntityUid uid, CargoOrderConsoleComponent com
                 return;
             }
 
+            // goob edit - resource siphon blocking access
+            var eqe = EntityQueryEnumerator<ResourceSiphonComponent>();
+            while (eqe.MoveNext(out var sip))
```

Comment:
```suggestion
            foreach (var sip in eqe)
```
due to above suggested change